scripts/build/build_libpq.sh

```diff
@@ -70,7 +109,55 @@ if [ "$ID" == "centos" ]; then
 fi
 
 
-if [ "$ID" == "centos" ]; then
+if [ "$ID" == "macos" ]; then
+
+    # Build kerberos if needed
+    krb5_dir="krb5-${krb5_version}/src"
+    if [ ! -d "${krb5_dir}" ]; then
+        "$curl" -sL \
+            curl -sL "https://kerberos.org/dist/krb5/$(echo 1.21.3 | grep -oE '\d+\.\d+')/krb5-${krb5_version}.tar.gz" \
+            | tar xzf -
+
+        cd "${krb5_dir}"
+
+        ./configure --prefix=${LIBPQ_BUILD_PREFIX} \
+            CPPFLAGS=-I${LIBPQ_BUILD_PREFIX}/include/ LDFLAGS=-L${LIBPQ_BUILD_PREFIX}/lib
+        make
+    else
+        cd "${krb5_dir}"
+    fi
+
+    make install
+    cd ../..
+
+fi
+
+
+if [ "$ID" == "macos" ]; then
+
+    # Build gettext if needed
+    gettext_dir="gettext-${gettext_version}"
```

Comment:
Which library requires gettext? Maybe we can just avoid it using `--without-gettext` or similar in `configure`.

Reply:
Hmm... thinking about it, the libpq does use localization. Better to have this then I guess.

Reply:
Still, I [run a test](https://github.com/psycopg/psycopg2/actions/runs/12622064372/job/35169563599) and it seems that gettext is not needed, at least on x86. @bwoodsend maybe this was needed in a different test?

Reply:
Well that's weird. It's providing `libintl` but the `otool -L` output says nothings linked against that library so I guess it is redundant. Can't think why I'd have thought that it was a dependency.

Reply:
Dropped, and it seems to work well on both architectures. Also re-enabled otool here. https://github.com/psycopg/psycopg2/actions/runs/12622655233